news/2023/2023-04-11-twin-trials-beatmap-voting-phase.md

```diff
@@ -0,0 +1,156 @@
+
+![Community Pick profile badge](/wiki/shared/news/2023-04-11-twin-trials-beatmap-voting-open/communitypick1.png) ![Community Pick profile badge](/wiki/shared/news/2023-04-11-twin-trials-beatmap-voting-open/communitypick2.png) ![Community Pick profile badge](/wiki/shared/news/2023-04-11-twin-trials-beatmap-voting-open/communitypick4.png) ![Community Pick profile badge](/wiki/shared/news/2023-04-11-twin-trials-beatmap-voting-open/communitypick3.png)
+
+Going forward, we'll require that contests have at least 20 participants to earn profile badge prizes. We'll specify this next time, but bypass it in this contest for the sake of mappers who already poured their souls into these submissions.
+
+### Twin Trials Champion
+
+Players have a chance to earn prizes here too!
+
+When the osu!(lazer) playlists close in two weeks, the players with the best scores in each playlist will receive these prizes:
+
+- **Easy/Normal/Hard/Insane playlist leaders:** 1 month of osu!supporter each
+- **Expert playlist leader:** 1 month of osu!supporter + *Twin Trials Champion* profile badge
+
+![Twin Trials Champion profile badge](/wiki/shared/news/2023-04-11-twin-trials-beatmap-voting-open/champion1.png) ![Twin Trials Champion profile badge](/wiki/shared/news/2023-04-11-twin-trials-beatmap-voting-open/champion2.png) ![Twin Trials Champion profile badge](/wiki/shared/news/2023-04-11-twin-trials-beatmap-voting-open/champion4.png) ![Twin Trials Champion profile badge](/wiki/shared/news/2023-04-11-twin-trials-beatmap-voting-open/champion3.png)
+
+Anyway, start voting already!
+
+## Submissions
+
+Click any banner below to download its team's mapset! Just keep in mind that you'll need to **play any difficulty from each submission in the osu!(lazer) Twin Trials playlists to be eligible to vote!**
+
+### osu!
+
+[Vote here!](https://osu.ppy.sh/community/contests/166)
+
+The **top 8 osu! submissions** were narrowed down for this vote:
+
+[![blacklist international team banner](/wiki/shared/news/2023-04-11-twin-trials-beatmap-voting-open/osu/blacklist%20international.png)](https://osu.ppy.sh/beatmapsets/1971997)
+
+[![bulochki k chayu team banner](/wiki/shared/news/2023-04-11-twin-trials-beatmap-voting-open/osu/bulochki%20k%20chayu.png)](https://osu.ppy.sh/beatmapsets/1972171)
+
+[![donerbuden bande team banner](/wiki/shared/news/2023-04-11-twin-trials-beatmap-voting-open/osu/donerbuden%20bande.png)](https://osu.ppy.sh/beatmapsets/1971983)
+
+[![Eye of Aeon team banner](/wiki/shared/news/2023-04-11-twin-trials-beatmap-voting-open/osu/Eye%20of%20Aeon.png)](https://osu.ppy.sh/beatmapsets/1972126)
+
+[![Hardvard Dropouts team banner](/wiki/shared/news/2023-04-11-twin-trials-beatmap-voting-open/osu/Harvard%20Dropouts.png)](https://osu.ppy.sh/beatmapsets/1972749)
+
+[![Impershable Night team banner](/wiki/shared/news/2023-04-11-twin-trials-beatmap-voting-open/osu/Imperishable%20Night.png)](https://osu.ppy.sh/beatmapsets/1972104)
```

Comment:
```suggestion
[![Impersihable Night team banner](/wiki/shared/news/2023-04-11-twin-trials-beatmap-voting-open/osu/Imperishable%20Night.png)](https://osu.ppy.sh/beatmapsets/1972104)
```

Reply:
"Imperishable" according to banner image file name